src/scrabble.js

```diff
@@ -1,5 +1,54 @@
-function scrabble() {
+function scrabble(word) {
+  //  Declare an object that shows all letters and their assigned numerical values.
+  const alphabet = {
```

Comment:
Alphabet is declared within the function, this is interesting because I thought it was supposed to be outside.

Reply:
good spot! I'll change that this afternoon :)

Reply:
creating variable inside the function very interseting